content/en/docs/setup/pick-right-solution.md

```diff
@@ -75,6 +75,8 @@ cluster.
 
 * [Kublr](https://kublr.com) offers enterprise-grade secure, scalable, highly reliable Kubernetes clusters on AWS, Azure, GCP, and on-premise. It includes out-of-the-box backup and disaster recovery, multi-cluster centralized logging and monitoring, and built-in alerting.
 
+* [KubeSail](https://kubesail.com) is an easy, free way to try Kubernetes.
```

Comment:
Free as in “tier”? I see there are [limits](https://kubesail.com/pricing).

Reply:
Hey thanks @sftim -- Yes, the free tier is limited, however what I'm trying to emphasize is that its a free tier with the lowest barrier to entry (a single click on the "Sign in with GitHub" button, and your namespace is created). All other free tiers that I'm aware of still require a credit card to get started.

In the spirit of keeping things simple, I trimmed down the wording as much as possible, but please feel free to revise the description to make it more clear.

Reply:
Seems like there genuinely is a free option here, so the wording you're proposed looks OK to me @pastudan